app/helpers/application_helper.rb

```diff
@@ -63,6 +63,10 @@ def completed_modules_table(modules)
     govuk_table(rows: [header, *rows], caption: 'Completed modules', first_cell_is_header: true)
   end
 
+  def module_completed_at(module_progress)
```

Comment:
Argument could be mad that the `strftime` be a method in `ModuleOverviewDecorator` and that it be passed to a locale, removing the need for a global helper.